src/main/kotlin/no/nav/familie/ks/sak/integrasjon/infotrygd/InfotrygdReplikaClient.kt

```diff
@@ -53,7 +53,7 @@ class InfotrygdReplikaClient(
             ) {
                 getForEntity(uri = requestURI)
             }
-        return barnMedLøpendeFagsak
+        return barnMedLøpendeFagsak.toSet()
```

Comment:
For meg gir det mest mening om denne metoden returnerer et `Set` fremfor `List`.